utils/build.ps1

```diff
@@ -1149,8 +1183,13 @@ function Get-Dependencies {
 
   if ($EnableCaching) {
     $SCCache = Get-SCCache
-    DownloadAndVerify $SCCache.URL "$BinaryCache\sccache-$SCCacheVersion.zip" $SCCache.SHA256
-    Expand-ZipFile sccache-$SCCacheVersion.zip $BinaryCache sccache-$SCCacheVersion
+    $FileExtension = if ($SCCache.URL.EndsWith('.tar.gz')) { "tar.gz" } else { "zip" }
+    DownloadAndVerify $SCCache.URL "$BinaryCache\sccache-$SCCacheVersion.$FileExtension" $SCCache.SHA256
+    if ($FileExtension -eq "tar.gz") {
+      Expand-TarBall sccache-$SCCacheVersion.$FileExtension $BinaryCache sccache-$SCCacheVersion
+    } else {
+      Expand-ZipFile sccache-$SCCacheVersion.$FileExtension $BinaryCache sccache-$SCCacheVersion
+    }
```

Comment:
Please file an issue upstream to provide the archive in a zipfile consistently for Windows.